src/openapi/.redocly.yaml

```diff
@@ -0,0 +1,31 @@
+# See https://docs.redoc.ly/cli/configuration/ for more information.
+
+organization: adobe
```

Comment:
Should this be "Adobe Developer" to match what shows int he Redocly dashboard?

Reply:
i used lower case because that is how it was used here: https://github.com/magento-devdocs/rest-api/blob/master/.redocly.yaml

should we capitalize this?

Reply:
according to the [docs](https://redocly.com/docs/cli/configuration/), it should be `adobe-developers`. but i don't think we're actually using it for anything.